### ISSUE TITLE
Issues with connections in Firefox

### ISSUE THREAD

**Issue by onel** [jeremija/peer-calls#32]
I seem to have a problem with Firefox, locally and on peercalls.com
It only appears between Firefox and Chrome.

If Chrome is the first one in the session then Firefox joins, sometimes I get this error in FF `Could not get access to microphone & camera` and sometimes the connection is never finished.
But if FF is first then Chrome, everything works.

I'm on Ubuntu, with:
Chrome: 57
Firefox: 52

Has anyone else seen this?

**Comment by onel** [comment]
Ok, there seems to be a problem in firefox with starting the camera. I don't know if this is a driver issue.
The error in `getCameraStream` is: `{message:"Starting video failed", name:"InternalError"}`

**Comment by jeremija** [comment]
Hi there, are you using Windows? I have the two browsers running simultaneously, one of them always disappears, but everything seems to work when I try it with two separate computers or computer & an android phone, or on my Linux box.

Unfortunately, I cannot reproduce the error you are getting (running Chrome 56 and FF 52).

**Comment by onel** [comment]
Hey, thanks for the response.
So I did some testing today and it seems to be related to my pc. This is what I tried:
Same pc ubuntu:
Firefox -> Firefox: works
Firefox -> Chrome:  doesn't work
Chrome -> Chrome: works
Chrome -> Firefox: works
Different pcs, ubuntu and windows, everything works except Firefox -> Chrome.

So it seems that only Firefox has the issue with accessing my video and audio if it is the first in the call, if it's the second one, I still get the error but I can see at least see the stream from Chrome.

Very strange and most likely something related to my ubuntu drivers.
Sorry for the false alarm.
